digdag-core/src/test/resources/io/digdag/core/workflow/parallel_limit.dig

```diff
@@ -0,0 +1,11 @@
++repeat:
+  loop>: 5
+  _parallel:
+    limit: 2
+  _do:
+    +step1:
+      echo>: '${this.task_name}'
+    +step2:
+      echo>: '${this.task_name}'
+    +step3:
+      echo>: '${this.task_name}'
```

Comment:
Please add a newline at the end of the string

Reply:
Fixed: cf56b4d